cmake/CPackOptions.cmake.in

```diff
@@ -42,13 +42,10 @@ elseif(CPACK_GENERATOR STREQUAL "RPM")
   set(CPACK_RPM_PRE_INSTALL_SCRIPT_FILE "${CMAKE_SOURCE_DIR}/cmake/CPack_preinstall.sh.in")
   set(CPACK_RPM_POST_INSTALL_SCRIPT_FILE "${CMAKE_SOURCE_DIR}/cmake/CPack_postinstall.sh.in")
   set(CPACK_RPM_PRE_UNINSTALL_SCRIPT_FILE "${CMAKE_SOURCE_DIR}/cmake/CPack_preuninstall.sh.in")
+  set(CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "/etc/profile.d/cubrid.csh /etc/profile.d/cubrid.sh /etc/init.d/cubrid /etc/init.d/cubrid-ha")
   set(CPACK_RPM_USER_FILELIST
     "%config(noreplace) /opt/cubrid/conf"
-    %dir "/opt/cubrid/databases"
-    "/etc/profile.d/cubrid.csh"
-    "/etc/profile.d/cubrid.sh"
-    "/etc/init.d/cubrid"
-    "/etc/init.d/cubrid-ha"
+    "%dir /opt/cubrid/databases"
```

Comment:
If you remove these files, problems may occur when installing with RPM packaging.
@kisoo-han 
Please give your opinion regarding this.